packages/components-react/src/examples/CarouselExampleFocusOnCenterSlide.tsx

```diff
@@ -40,6 +40,10 @@ export const CarouselExampleFocusOnCenterSlidePage = (): JSX.Element => {
       heading="Some Heading"
       onUpdate={(e) => onCarouselUpdate(e.detail.activeIndex)}
       focusOnCenterSlide
+      trimSpace={false}
+      gradientColor="background-surface"
+      // @ts-expect-error: ignore unknown property (TS2353)
```

Comment:
```suggestion
```

Shouldn't be necessary

Reply:
In my IDE (WebStorm) i get this:

<img width="1271" alt="image" src="https://github.com/user-attachments/assets/9e864d00-5e04-4c88-be1d-bbc1be2ab5db">